scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -104,9 +106,34 @@ class RedundantBraces(implicit ctx: RewriteCtx) extends RewriteSession {
       case t: Term.Interpolate if settings.stringInterpolation =>
         processInterpolation(t)
 
+      case Importer(_, List(importee)) if ctx.dialect.allowAsForImportRename =>
+        processSingleImport(importee)
+
       case _ =>
     }
 
+  private def processSingleImport(tree: Importee) = {
+    val tokens = tree.tokens
+    tree match {
+      case _: Importee.Rename | _: Importee.Unimport
+          if !tokens.exists(_.is[RightArrow]) =>
```

Comment:
should this be a positive check rather than negative? i.e., look for `Ident("as")` explicitly?

Reply:
That is the magic of soft keywords, you can actually do:
`import as as as` or `import a.{ as => as }`

and it compiles :D

No `=>` is a safer choice.